<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stepper and DC Motor Controller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>UART Settings</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Device select:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxUartDeviceSelect"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUartConnect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Co&amp;nnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUartReload">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Reload</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidgetDeviceSelection">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabSteppers">
       <attribute name="title">
        <string>Stepper</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidgetStepperSelect">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDCMotors">
       <attribute name="title">
        <string>DC Motor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidgetDCMotorSelect">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionStepper"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Author"/>
    <addaction name="action_About_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuSettings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionStepper">
   <property name="text">
    <string>Stepper</string>
   </property>
  </action>
  <action name="action_Author">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
